archive/s/scala/.gitignore

```diff
@@ -0,0 +1 @@
+*.txt
```

Comment:
What's this .gitignore for?

Reply:
Shoot! I was using that to exclude txt files as input output from my git diff. I will make another change that take it out.